lib/cloudstore/rate_limiter.es6.js

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  package: 'org.chromium.apis.web',
+  name: 'RateLimiter',
+  documentaion: `Limit request rate.`,
```

Comment:
Should component-level documentation contain information about default values, like `interval=50`?